src/main/java/me/twodee/quizatron/Main.java

```diff
@@ -14,10 +14,12 @@
 import javafx.scene.text.Font;
 import javafx.stage.Stage;
 
-import static com.sun.javafx.scene.control.skin.Utils.getResource;
+import java.util.logging.Logger;
 
 public class Main extends Application {
 
+    private static final Logger log = Logger.getLogger(Main.class.getName());
```

Comment:
Hi, thanks for your contributions. Given that we aren't logging anything yet, please explain why adding a logger might be beneficial.

Reply:
Before, this exception prevented application to start. It should start even there is no font and we cant hide problem.